benchmarks/utils/join.py

```diff
@@ -1,28 +1,117 @@
 import pandas as pd
+import os
+from pathlib import Path
 
 
-def fetch_data(dataset_name):
+def get_local_data(dataset_name):
+    """ Get the path to the local datasets. """
+    module_path = Path(os.path.dirname(__file__)).resolve()
+    data_dir = module_path / "data"
```

Comment:
Why not use `skrub.utils.get_data_dir` like `get_data_dir("benchmarks_data")`?
I think saving the data inside the `utils` folder is not appropriate.

Reply:
Good idea